### ISSUE TITLE
Assess damage from accidental publish to phet.colorado.edu

### ISSUE THREAD

**Issue by zepumph** [phetsims/circuit-construction-kit-black-box-study#65]
Last night I was trying to publish CCK BBS 1.1.4 to `phet-io.colorado.edu` but accidentally left out the `--brand=phet-io`. @mattpen what needs to be done with the copy on `phet.colorado.edu`?  It isn't public (except that the index directories are).

**Comment by mattpen** [comment]
In addition to the documents being published at https://phet.colorado.edu/sims/html/circuit-construction-kit-black-box-study/1.1.4/, there is also an entry for the sim in the database and the sim is available in rosetta.

I can remove the files from the server and purge the database entry.  I think there is a file in rosetta that may need to be changed as well.

**Comment by mattpen** [comment]
We'll need to purge the entry from https://github.com/phetsims/rosetta/blob/master/data/simInfoArray.json as well.  @jbphet - Do you think it will be ok to just manually delete it and push the update?  Would we need to restart rosetta afterwards?

**Comment by mattpen** [comment]
The sim files and DB entry have been removed.

**Comment by jbphet** [comment]
I manually removed this sim from the `simInfoArray` file, which is where rosetta gets the list of active simulations.  The then verified that this removes CCK Black Box Study from the list of translatable simulations.

I checked rosetta's log, and it looks like some translators messed around with this simulation, but none of them submitted a translation, so I don't need to remove anything from Babel.  As long as the work that @mattpen did prevents this sim from reappearing in simInfoArray, the translation portion of this issue should now be resolved.
